tests/test_compile.py

```diff
@@ -155,3 +158,47 @@ def test_compile__compile_restricted_eval__used_names(c_eval):
     assert result.errors == ()
     assert result.warnings == []
     assert result.used_names == {'a': True, 'b': True, 'x': True, 'func': True}
+
+
+@pytest.mark.parametrize(*c_single)
+def test_compile__compile_restricted_csingle(c_single):
+    """It compiles code as an Expression."""
+    result = c_single('4 * 6')
+    assert result.code is None
+    assert result.errors == (
+        'Line None: Interactive statements are not allowed.',
+    )
+
+
+PRINT_EXAMPLE = """
+def a():
+    print 'Hello World!'
+"""
+
+
+@pytest.mark.skipif(
+    IS_PY3,
+    reason="Print statement is gone in Python 3."
+           "Test Deprecation Warming in Python 2")
+def test_compile_restricted():
+    with pytest.warns(SyntaxWarning) as record:
+        result = compile_restricted(PRINT_EXAMPLE, '<string>', 'exec')
+        assert isinstance(result, types.CodeType)
+        assert len(record) == 2
+        assert record[0].message.args[0] == \
+            'Line 3: Print statement is deprecated ' \
+            'and not avaliable anymore in Python 3.'
+        assert record[1].message.args[0] == \
+            "Line 2: Prints, but never reads 'printed' variable."
+
+
+EVAL_EXAMPLE = """
+def a():
+    eval('2 + 2')
+"""
+
+
+def test_compile_restricted_eval():
+    with pytest.raises(SyntaxError,
+                       message="Line 3: Eval calls are not allowed."):
+        compile_restricted(EVAL_EXAMPLE, '<string>', 'exec')
```

Comment:
This is already tested in `test_transformer.test_transformer__RestrictingNodeTransformer__visit_Call__3`. Or am I missing a difference here between those two tests?